ArduCopter/AP_Arming.cpp

```diff
@@ -49,7 +49,7 @@ bool AP_Arming_Copter::run_pre_arm_checks(bool display_failure)
         return mandatory_checks(display_failure);
     }
 
-    return parameter_checks(display_failure)
+    return uint8_t(parameter_checks(display_failure))
```

Comment:
This deserves a comment.  Otherwise we'll be bouncing PRs to turn this back into a bool (or, worse, merging them!)

Reply:
Fixed